lib/Sema/DerivedConformanceAdditiveArithmeticVectorNumeric.cpp

```diff
@@ -157,8 +156,8 @@ bool DerivedConformance::canDeriveAdditiveArithmetic(NominalTypeDecl *nominal,
       C.getLazyResolver()->resolveDeclSignature(v);
     if (!v->hasInterfaceType())
       return false;
-    auto declType = DC->mapTypeIntoContext(v->getValueInterfaceType());
-    return (bool)TypeChecker::conformsToProtocol(declType, addArithProto, DC,
+    auto varType = DC->mapTypeIntoContext(v->getValueInterfaceType());
```

Comment:
Does this variable ```varType```` get modified?

Reply:
No, it does not get modified. Are you suggesting that I should skip the definition of `varType`?
I would actually prefer to keep the local variable definition for clarify/consistency, if you don't mind.

Reply:
I was just gunna suggest to make is ```const``` if it doesn't get modified.

Reply:
It is well-established local convention to _not_ use const for local variables.